### ISSUE TITLE
Consider using `-i` instead of configuring ssh in the local machine for linux

### ISSUE THREAD

**Issue by odarbelaeze** [aiidateam/aiida-tutorials#47]
```
ssh -i /path/to/key-max.pem -L 8888:localhost:8888 -X -C max@18.203.248.68
```

Might do the trick as a one command and go option.

**Comment by yakutovicha** [comment]
I would actually propose the inverse solution. Why not to put everything in `~/.ssh/config`?
```
Host aiidatutorial
  Hostname 34.242.168.15
  User max 
  IdentityFile ~/.ssh/aiida-tutorial-max.pem
  ForwardX11 yes
  ForwardX11Trusted yes
  LocalForward 8888 localhost:8888
```
and do not explain why -X or -Y is sometimes needed. It should just work like this:
```
$ ssh aiidatutorial
```

**Comment by odarbelaeze** [comment]
@yakutovicha I like your thinking there, it would simplify the config instructions, and while we're at it, we can also add the command to do the same,

```bash
$ ssh \
    -i ~/.ssh/aiida-tutorial-max.pem \
    -L 8888:localhost:8888 \
    -X -C \
     max@34.242.168.15
```

I know at least I would appreciate, copy and paste the command, let me take care of updating the instructions.

**Comment by ltalirz** [comment]
actually, I just checked and this fixes #56 as well.
I will update the section, adding the lines to the ssh config + potentially also the 1-liner.

**Comment by ltalirz** [comment]
ah - I see @odarbelaeze is working on it.
nevermind - go ahead please!

**Comment by odarbelaeze** [comment]
@ltalirz next time I'll remember to assign it to me after I get the branch going, I'll take care of it in the morning.

**Comment by ltalirz** [comment]
fixed in #83
